specs/flyout/flyout.md

```diff
@@ -0,0 +1,174 @@
+# Popover (aka Flyout)
+## Overview
+Popover is a focusable floating container that shows over a pages content and displays UI related to what the user is doing and is typically triggered by a clickable element. Similar to a Dialog, but with positioning controls, no overlay option, and no specific action is required, a Popover can be used to reveal a secondary control or show more detail about an item.
+
+### Background
+A Popover can be less intrusive to the user experience than a Dialog and should be used instead of a Dialog when information or input is needed to be displayed, but not in the center of the screen, such as in the case of collection of additional info before an action, displaying information that is only relevant some of the time, or just displaying more information.
```

Comment:
```suggestion
A Popover can be less intrusive to the user experience than a Dialog and should be used instead of a Dialog when information or input is needed to be displayed, but not in the center of the screen, such as in the case of the collection of additional info before an action, displaying information that is only relevant some of the time or for just displaying more information.
```